src/components/CategoryNavBar.jsx

```diff
@@ -5,6 +5,10 @@ import { useEffect, useState } from "react"
 const CategoryNavBar = () => {
     const [categories, setCategories] = useState([])
 
+    const formatWord = (y) => {
```

Comment:
Cool util function 😎 Using y as the parameter name isn't overly clear what this value is so it would be good to make this a little more descriptive, you could also extract this out to a utils file if you wanted to keep the component slightly cleaner.